tasks.py

```diff
@@ -286,7 +286,7 @@ def sharejs(host=None, port=None, db_host=None, db_port=None, db_name=None, cors
 @task(aliases=['celery'])
 def celery_worker(level="debug"):
     """Run the Celery process."""
-    cmd = 'celery worker -A framework.tasks -l {0}'.format(level)
+    cmd = 'celery worker -A framework.tasks -l {0} --beat'.format(level)
```

Comment:
Commented in code.

This is replacing a cronjob that would run once every 5 minutes and once daily. The current cronjob that is replaced runs once daily.